code/modules/antagonists/heretic/status_effects/mark_effects.dm

```diff
@@ -260,3 +260,50 @@
 /datum/status_effect/eldritch/lock/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_ALWAYS_NO_ACCESS, STATUS_EFFECT_TRAIT)
 	return ..()
+
+// MARK OF MOON
+
+/datum/status_effect/eldritch/moon
+	effect_icon_state = "emark8"
+	///Used for checking if the pacifism effect should end early
+	var/damage_sustained = 0
+
+/datum/status_effect/eldritch/moon/on_apply()
+	.=..()
+	ADD_TRAIT(owner, TRAIT_PACIFISM, id)
+	owner.emote(pick("giggle", "laugh"))
+	owner.balloon_alert(owner, "you feel unable to hurt a soul!")
+	RegisterSignal (owner, COMSIG_MOB_APPLY_DAMAGE, PROC_REF(on_damaged))
+	return TRUE
+
+/// Signal proc for [COMSIG_MOB_APPLY_DAMAGE], being damaged past a threshold will remove the PACIFISM trait
```

Comment:
but like why tho. where it comes from is told by the proc name, I'd like to know why it's doing what it's doing